resources/helm-dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM docker.io/alpine/helm:3.15.3
```

Comment:
What is this for?

Reply:
This is a bless! :-) 
I was about to ask @eloycoto to make his magic and make something similar to the container for generating the docs.

This one generates the values scheme json, which I used to do manually and it was tedious task after each change to the values.yaml.

Now this can be simplified by running
`make generate-docs` and getting both README.md and values.schema.json updated according to the changes introduced to the values.yaml.

Thank you, Eloy!

Reply:
Let me compare with the result of how I usually creates it.

Reply:
@jordigilh you are correct, there are major differences.
The `values.schema.json` generated by using the website is > 1400 lines vs the `values.schema.json` created by the container (~400 lines).

See [1] https://pastebin.com/ET7AAbcF 
I was hoping to reduce the annoying step of doing manually.
@eloycoto pls see the difference in the output between [1] that was generated by https://www.jsonschema.net/app/schemas/470628 to the existing file.